### DISCUSSION TITLE
launch program on key release

### DISCUSSION THREAD

**Discussion by aloispichler** [qtile/qtile#3918] [Q&A]
I'd like to launch rofi, when the modkey (winkey) is _released_.
For that, the code in _i3/config_ is
`bindcode 133 --release exec --no-startup-id rofi -show filebrowser`.
How can this be accomplished in qtile?
Thanks in advance!

**Comment by elParaguayo** [comment]
We don't support keybindings on key release yet. However, there is an open PR for this (#2785) which we could look at again.
